test/e2e/dra/dra.go

```diff
@@ -1499,9 +1500,37 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), feature.Dynami
 			// Delete the first pod
 			b.deletePodAndWaitForNotFound(ctx, pod)
 
-			// There shoud not be available devices for pod2.
+			// There shoud be available devices for pod2.
 			b.testPod(ctx, f, pod2)
 		})
+
+		f.It("must assign devices to nodes", func(ctx context.Context) {
+			nodes := NewNodesNow(ctx, f, 2, 2)
+			driver := NewDriverInstance(f)
+			driver.Run(nodes, resourcesWithPerDeviceNodeSelection(nodes))
+			b := newBuilderNow(ctx, f, driver)
+			// The first pod will use one of the devices. Since both devices are
+			// available, there should be sufficient counters left to allocate
+			// a device.
+			claim := b.externalClaim()
+			pod := b.podExternal()
+			pod.Spec.ResourceClaims[0].ResourceClaimName = &claim.Name
```

Comment:
Is this line really needed? I think the default should just work when we only have a single ResourceClaim and pod in the test.

Reply:
It should be set correctly by default. It is just some scenarios where we need to update it.